test/fixtures/operatingsystems.yml

```diff
@@ -64,4 +64,24 @@ coreos:
   release_name: stable
   architectures: x86_64
   media: coreos
-  title: 'CoreOS 494.5.0'
\ No newline at end of file
+  title: 'CoreOS 494.5.0'
+
+debian7_0:
```

Comment:
Prefer adding new Factories or using existing ones, we are trying to reduce fixtures usage as much as possible.